src/hub/routes/firefox.py

```diff
@@ -23,11 +23,12 @@ def route(self) -> None:
             sns_client = boto3.client("sns", region_name=CFG.AWS_REGION)
             response = sns_client.publish(
                 TopicArn=CFG.TOPIC_ARN_KEY,
-                Message=json.dumps({"default": json.dumps(self.payload)}),
+                Message=json.dumps({"default": self.payload}),
```

Comment:
this means that we will be passing json encoded strings, right?  why?

Reply:
This is a requirement from FxA.

Reply:
Let's add a comment inline with the code saying that this is a requirement from them then; will remind us in the future

Reply:
Done.